client/pom.xml

```diff
@@ -38,7 +38,7 @@
            <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
```

Comment:
Because this client is generated from the Conjur OpenAPI description, these versions have to be updated in this file's template as well - can be found at `templates/libraries/okhttp-gson/pom.mustache`. Otherwise, these updates will be wiped out the next time the OpenAPI description changes.